operator/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## Main
 
 - [6195](https://github.com/grafana/loki/pull/6195) **periklis**: Add ruler config support
+- [6288](https://github.com/grafana/loki/pull/6288) **aminesnow**: Expose only an HTTPS gateway when in openshift mode
```

Comment:
Reminder before we squash & merge: We generally add for every new PR the changelog entry at the top. I believe yours got pushed down through rebasing onto main.

Reply:
indeed